dali/operators/audio/resample.cc

```diff
@@ -96,6 +96,17 @@ type. Example::
 )",
     nullptr, false);
 
+// Deprecated alias
+DALI_SCHEMA(experimental__AudioResample)
+    .AddParent("AudioResample")
+    .NumInput(1)
+    .NumOutput(1)
+    .Deprecate(
+        "AudioResample",
+        R"code(This is just an deprecated alias kept for backward compatibility.)code");  // Deprecated
```

Comment:
How about adding information that we moved it out of experimental? If you can find better words to describe it, feel free to do it:
```suggestion
        R"code(This operator was moved out from the experimental phase, and is now regular DALI operator. This is just an deprecated alias kept for backward compatibility.)code");  // Deprecated
```